labs/lab0/chapter0.md

```diff
@@ -51,141 +49,80 @@ $ cd ~/
-`vagrant halt`. To destroy the machine, you can run `vagrant destroy`.
+$ minishift status
+Running
 ```
 
 Now we can actually step inside the machine with:
 
 ```bash
-$ vagrant ssh
-[vagrant@rhel-cdk ~]$ 
-```
-
-Now take a look at `ip -4 -o addr` and see what IPs we have: 
-
-```bash
-[vagrant@rhel-cdk ~]$ ip -4 -o addr
-1: lo    inet 127.0.0.1/8 scope host lo\       valid_lft forever preferred_lft forever
-2: eth0    inet 192.168.121.170/24 brd 192.168.121.255 scope global dynamic eth0\       valid_lft 3430sec preferred_lft 3430sec
-3: eth1    inet 10.1.2.2/24 brd 10.1.2.255 scope global eth1\       valid_lft forever preferred_lft forever
-4: docker0    inet 172.17.0.1/16 scope global docker0\       valid_lft forever preferred_lft forever
+$ minishift ssh
+Last login: Mon Feb 30 17:49:01 2017 from 192.168.??.??
+[docker@minishift ~]$ 
 ```
 
-The `10.1.2.2` address is the one we set in the `Vagrantfile`. If you
-peak in the file you'll see it set at the top in a variable: 
-`PUBLIC_ADDRESS="10.1.2.2"`. We'll use the `10.1.2.2` address to
-access the CDK in the next section.
+You should very rarely need to jump inside the VM as most of the functions of docker and OpenShift can be done remotely. However, it can be really nice to know that you don't need to figure out the IP address or the username and password in case you have to get in there when something goes wrong. That said, most of the time the right answer is to just destroy the instance and recreate it.
 
-Now exit out of the CDK VM by disconnecting from the SSH session:
+Now exit out of the minishift VM by disconnecting from the SSH session:
 
 ```bash
 $ exit
 ```
 
+We have two more commands worth mentioning. First off, let's mention `minishift stop`. Stop does exactly what it sounds like and shuts down minishift. However, it does not destroy anything inside just "turns the machine off." If you do want to remove the VM, you can use `minishift delete`. You can spin it right back up again, fresh, with `minishift start`. Finally, we will use `minishift docker-env` in a few minutes to connect the host to the docker daemon in the VM.
+
 ## Container Development Kit (CDK) Walkthrough
 
-The cool thing is, we actually launched the CDK VM by executing `vagrant up` on
-the Vagrantfile in the previous section. The CDK gives a couple options (through the use of 
-[different Vagrantfiles](https://developers.redhat.com/download-manager/file/cdk-2.0.0.zip)) 
-but in this case, we launched the "OpenShift in a Box" VM (aka rhel-ose in 
-the cdk.zip). The CDK provides a simple to use and launch instance of the 
+When we started minishift, we launched the software tooling component of the CDK. As we said before, the CDK provides a lot of support for containerizing your applications. However, the major software tool is minishift.
+
+However, what is minishift? Essentially, it is a simple to use and launch instance of the 
 same OpenShift PaaS you would use at work. Why is a PaaS included in a tool, 
```

Comment:
i wonder if we should avoid calling it a PaaS at this point. It's more than that :)